volumes/frus1969-76ve03.xml

```diff
@@ -3830,12 +3830,14 @@
                                     target="#t_US1">U.S.</gloss> delegation to the 1973 <gloss
                                     target="#t_UN1">UN</gloss> Seabed Committee meeting.</note>
                         </head>
-                        <dateline>
-                            <date notBefore="1972-12-18T00:00:00-05:00"
-                                notAfter="1973-03-16T23:59:59-05:00"
-                                ana="#date_undated-inferred-from-document-content"
-                                calendar="gregorian">undated</date>
-                        </dateline>
+                        <opener>
+                            <dateline>
+                                <date notBefore="1972-12-18T00:00:00-05:00"
+                                    notAfter="1973-03-16T23:59:59-05:00"
+                                    ana="#date_undated-inferred-from-document-content"
+                                    calendar="gregorian">undated</date>
+                            </dateline>
```

Comment:
Are brackets needed around `<date>` here? (I noticed brackets being added to `<date ana="#date_undated...">` elsewhere in this PR and am guessing at if adding brackets is a new guideline?)

Reply:
(Same question as above for the other `<date>` elements here.)

Reply:
I added brackets if the text was inserted post-publication.  If the document had been published with undated in the dateline, I didn't add the brackets.  For electronic-only volumes, I'll go ahead and bracket the undated.

Reply:
Thanks for clarifying this. As mentioned by email, please hold off on adding brackets to the electronic-only volumes for the moment. I'll confirm the plan for these.